src/controllers/user.controllers.js

```diff
@@ -6,17 +6,16 @@ import { editValidation, loginValidation, passwdEditValidation, registerValidati
 import { getByIdAbstract } from '../services/dbMethods';
 import userUpdated from '../services/userUpdated';
 
+const USER_ACTIVE = 1;
```

Comment:
I think this could possibly go into a separate file in another folder, for instance in the folder `constants`. Please apply this in every place that currently uses a number for status and create a constant for `USER_DELETED` and do the same with it.